#collison message

string uav_name
int8 front
int8 back
int8 left
int8 right 
int8 bottom

